### DISCUSSION TITLE
Backup within home assistant

### DISCUSSION THREAD

**Discussion by rizzol19** [babybuddy/babybuddy#653] [General]
Not sure if this is a general home assistant issue, but I want to move my home assistant instance from virtual box to VMware. I made a complete backup however baby buddy which is running within home assistant comed up as a fresh install on the new instance after restoring backup.  Any help with using my current baby buddy info would be great. Thanks!

**Comment by cdubz** [comment]
I'm not sure how a backup like this would be handled. Can you clarify exactly what you mean by a "complete backup"? What mechanism did you use to backup and restore the data?

@OttPeterR anything to add here? E.g. about how data for Baby Buddy is stored/backed up (or not) in HA with the addon?

**Comment by OttPeterR** [comment]
There's some reading [here](https://www.home-assistant.io/common-tasks/os/#backups) on how backups work, but at a high level it's docker doing a snapshot and restore of the important directories the add-on uses. I've had successful backups and restores on my own hardware as well as a migration from armv7 to aarch64, but not the way you're doing it. Could you try making a backup of just baby buddy and try restoring that?

**Comment by rizzol19** [comment]
My novice might be showing here but I did a complete backup from within the supervisor menu. Then when I started vm ware/new home assistant instance I restored from there instead of creating a new one. Is there any other way to do it?

**Comment by OttPeterR** [comment]
You can continue with your newly created Home Assistant, just go back to the first HA and in the Backups menu, you can make a partial backup of just Baby Buddy and try restoring that one

**Comment by rizzol19** [comment]
I did just that and it worked perfect. Thank you!
